Frontend/android-native/mobility-common/src/main/java/in/juspay/mobility/common/MobilityCommonBridge.java

```diff
@@ -1656,6 +1677,69 @@ protected Bitmap getMarkerBitmapFromView(String locationName, String imageName,
         return returnedBitmap;
     }
 
+    protected void onMapUpdate() {
+        try {
+            MapRemoteConfig mapRemoteConfig = getMapRemoteConfig();
+            if(mapRemoteConfig.enableMapRecenter) {
+                String merchant = bridgeComponents.getContext().getResources().getString(in.juspay.mobility.app.R.string.service);
```

Comment:
Is it compiling? I think it is not rebased. can you check that. you can't access the `mobility-app` inside `mobility-common`.